moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -188,11 +191,10 @@ struct FCLGeometry
   }
 
   std::shared_ptr<fcl::CollisionGeometry> collision_geometry_;
-  boost::shared_ptr<CollisionGeometryData>  collision_geometry_data_;
+  CollisionGeometryDataPtr collision_geometry_data_;
 };
 
-typedef std::shared_ptr<FCLGeometry> FCLGeometryPtr;
-typedef std::shared_ptr<const FCLGeometry> FCLGeometryConstPtr;
+MOVEIT_CLASS_FORWARD(FCLGeometry);
```

Comment:
oh, then why did you convert it recently to std?
https://github.com/delftrobotics/moveit/commit/735249528b2c1e3d77edc64cbff2d9d1c7f40025#diff-90ab8a9db10c5c40f0754a0638741a07L193